node/cli/src/command.rs

```diff
@@ -133,29 +134,33 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn ChainSpec>, String> {
 			if ChainIdentity::Frequency == spec.identify() {
 				#[cfg(feature = "frequency")]
 				{
-					Box::new(chain_spec::frequency::ChainSpec::from_json_file(path_buf)?)
+					return Ok(Box::new(chain_spec::frequency::ChainSpec::from_json_file(path_buf)?))
```

Comment:
Homogenize all arms of match to do a return, that way our match arms are not divergent